### PR TITLE
Add more static visual tests for amphtml-ads

### PR DESCRIPTION
**Pull request by powerivq**
Move these resources from examples to examples/visual-tests for awareness of that the file will affect tests.

### CODE REVIEW

File: test/visual-diff/visual-tests [hunk 1]
```diff
@@ -33,6 +33,7 @@
   "asset_globs": [
     "examples/visual-tests/**",
     "examples/amphtml-ads/ads-tag-integration.js"
```
Comment (estherkim) [inline test/visual-diff/visual-tests]: missing , at the end

Comment (lannka) [reply test/visual-diff/visual-tests]: hmm, shall we run lint on this file?

Comment (powerivq) [reply test/visual-diff/visual-tests]: I want to see it fails the job at the very least.

Comment (lannka) [reply test/visual-diff/visual-tests]: yep. lint is a easier approach to catch this mistake
surfacing a proper error is going to be more helpful in long run.

I think we should do both

**Pull request closed without merging**